src/qtt/data.py

```diff
@@ -1029,183 +1031,326 @@ def saveExperimentData(outputdir, dataset, tag, dstr):
     write_data(path, dataset)
 
 
-def makeDataSet1Dplain(xname, x, yname, y=None, xunit=None, yunit=None, location=None, loc_record=None):
-    ''' Make DataSet with one 1D array and one setpoint array
+def determine_parameter_unit(parameter):
+    """ Determine unit associated with a qcodes.parameter, otherwise None """
```

Comment:
No args documentation?

Reply:
determine_parameter_unit was an existing method. I now added docstrings for it and added 100% unit tests for it.